src/sys/timerfd.rs

```diff
@@ -1,33 +1,45 @@
 //! Timer API via file descriptors.
 //!
-//! Timer FD is a Linux-only API to create timers and get expiration notifications through file
-//! descriptors.
+//! Timer FD is a Linux-only API to create timers and get expiration
+//! notifications through file descriptors.
 //!
 //! For more documentation, please read [timerfd_create(2)](http://man7.org/linux/man-pages/man2/timerfd_create.2.html).
 //!
 //! # Examples
 //!
-//! Create a new one-shot timer that expires after 5 seconds.
-//! ```no_run
-//! # use nix::sys::timerfd::{TimerFd, ClockId, TimerFlags, TimerSetTimeFlags, Expiration};
+//! Create a new one-shot timer that expires after 1 second.
+//! ```
+//! # use std::os::unix::io::AsRawFd;
+//! # use nix::sys::timerfd::{TimerFd, ClockId, TimerFlags, TimerSetTimeFlags,
+//! #    Expiration};
 //! # use nix::sys::time::{TimeSpec, TimeValLike};
+//! # use nix::unistd::read;
 //! #
 //! // We create a new monotonic timer.
-//! let fd = TimerFd::new(ClockId::Monotonic, TimerFlags::empty()).unwrap();
+//! let fd = TimerFd::new(ClockId::CLOCK_MONOTONIC, TimerFlags::empty())
+//!     .unwrap();
 //!
 //! // We set a new one-shot timer in 5 seconds.
-//! fd.settime(TimerSetTimeFlags::empty(), Expiration::OneShot(TimeSpec::seconds(5))).unwrap();
+//! fd.settime(
+//!     Expiration::OneShot(TimeSpec::seconds(5)),
```

Comment:
Be sure to set the time to just 1 second.  5 seconds is too long for a doc test.

Reply:
Ok, fixed it.